pkg/features/versioned_kube_features.go

```diff
@@ -183,7 +183,8 @@ var defaultVersionedKubernetesFeatureGates = map[featuregate.Feature]featuregate
 	},
 
 	ExecProbeTimeout: {
-		{Version: version.MustParse("1.20"), Default: true, PreRelease: featuregate.GA}, // lock to default and remove after v1.22 based on KEP #1972 update
+		{Version: version.MustParse("1.20"), Default: true, PreRelease: featuregate.GA},
```

Comment:
ExecProbeTimeout was GA in 1.20. Even if https://github.com/kubernetes/enhancements/pull/3201 re-promoted it to GA in 1.24, the feature-gate is still GA since 1.20. Do we need to add a new item for this KEP changes?